birdhouse/env.local.example

```diff
@@ -476,6 +476,40 @@ export THREDDS_ADDITIONAL_CATALOG=""
 #    </datasetScan>
 #"
 
+# Additional file filters to add for the Service Data THREDDS dataset. By default, the Service Data dataset will only
+# serve files with the following extensions: .nc .ncml .txt .md .rst .csv
+# If you need this dataset to serve other files you should update the THREDDS_SERVICE_DATA_EXTRA_FILE_FILTERS to add
+# additional file filters.
+# This may be useful to set if a WPS outputs files to the wps_outputs/ directory (hosted under the Service Data dataset) 
+# in a file format other than one of the defaults.
+# See the example below which would also enable serving .png and .h5 files.
+#export THREDDS_SERVICE_DATA_EXTRA_FILE_FILTERS="
```

Comment:
The example here should be using single quote so the double quote inside do not have to be escaped, same as in the `default.env` file.